examples/create-react-app/src/index.js

```diff
@@ -15,7 +23,8 @@ if (!baseUrl) {
 init({
     baseUrl: baseUrl + '/api/29'
 }).then(d2 => {
-    console.log(d2)
+    console.log(d2);
```

Comment:
Perhaps remove this log

Reply:
Done. I have to admit I tend also to add those `console.log(d2)` / `window.d2 = d2` to work with d2 directly from the console.